icu4c/source/configure.ac

```diff
@@ -1191,6 +1191,7 @@ case "${host}" in
 	*-*-nto*)	platform=U_QNX ;;
 	*-dec-osf*) 	platform=U_OSF ;;
 	*-*-beos)		platform=U_BEOS ;;
+	*-*-haiku)	platform=U_HAIKU ;;
```

Comment:
OK, a bit wondering here, I see Darwin for instance also using U_DARWIN but not U_PF_DARWIN as in the rest of the code, I'm guessing U_HAIKU can stay as it is atm?

Reply:
I only see `platform` used in `config/gmakever.mk` even though it is substituted later in this file.

Reply:
There is no source code (.h, .c, .cpp) that uses U_DARWIN. It's possible that this platform variable is a leftover from when platform.h didn't do as much fancy footwork.

Reply:
FYI I filed ICU-22918 for this issue.